Gruntfile.js

```diff
@@ -0,0 +1,34 @@
+module.exports = function (grunt) {
```

Comment:
As I explained, this is the Javascript equivalent of a ``Makefile``.  Currently, we're only doing one thing, which is using the ``grunt-bower-concat`` library to condense all of our Javascript dependencies into a single JS file and a single CSS file.

Theoretically we could minify those files as well if we wanted.  I don't think that's worth the hassle but it might be worth measuring the performance difference at some point just to see.  I doubt we'd see any perceptible gain, but you never know.